kong.conf.default

```diff
@@ -137,6 +137,9 @@
                                                             # dump info of all plugins it
                                                             # manages
 
+#protobuf_includes = /usr/include   # Comma-separated list of directories to search for
```

Comment:
unrelated but does it make sense to reuse this filed in grpc plugins to resolve issues like https://github.com/Kong/kong-plugin-grpc-gateway/issues/15?

Reply:
unfortunately, that's not enough to make `import` work transparently in procfiles.  even with the `p.include_imports = true` line, lproc only parses those imported files, they're not loaded into the codec context.  That's why a few lines down I added explicit `p:loadfile()` lines for empty.proto and struct.proto.   I think `import public ...` should be enough, but seems something is missing.